recipe/meta.yaml

```diff
@@ -8,118 +8,71 @@ package:
-    build:
-      script:
-        - python -m pip install . --no-deps -vv
-      entry_points:
-        - pytensor-cache = bin.pytensor_cache:main
-    requirements:
-      build:
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - python                                 # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - cython                                 # [build_platform != target_platform]
-        # Allow numpy pinnings to be managed by conda-forge global dependencies
-        - numpy                                  # [build_platform != target_platform]
-        - versioneer ==0.28                      # [build_platform != target_platform]
-        - tomli                                  # [build_platform != target_platform]
-        - setuptools >=48.0.0                    # [build_platform != target_platform]
-      host:
-        - pip
-        - python
-        - cython
-        # Allow numpy pinnings to be managed by conda-forge global dependencies
-        - numpy
-        - versioneer ==0.28
-        - tomli
-        - setuptools >=48.0.0
-      run:
-        - python
-        # Ref: <https://github.com/pymc-devs/pytensor/blob/rel-{{ version }}/setup.py>
-        # numpy global dependency is greater than 1.17 on conda-forge
-        # so let conda-forge manage it.
-        - {{ pin_compatible("numpy") }}
-        - scipy >=0.14
-        - filelock
-        - etuples
-        - logical-unification
-        - miniKanren
-        - cons
-        - typing_extensions
-        - setuptools >=48.0.0
-    test:
-      imports:
-        - pytensor
-      commands:
-        - pytensor-cache help
-        - pip check
-        - python check-for-warnings.py allowed-warnings-base.txt
-      requires:
-        - pip
-      files:
-        - check-for-warnings.py
-        - allowed-warnings-base.txt
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - pip
+    - numpy {{ numpy }}
+    - versioneer 0.28
+    - tomli
+    - setuptools >=48.0.0
+    # mkl-service is only available for linux-64, win-64, and osx-64.
+    # For everything else, use openblas.
+    - mkl-service  # [not ((osx and x86_64) or linux and (aarch64 or ppc64le))]
+    - openblas  # [(osx and x86_64) or linux and (aarch64 or ppc64le)]
+    # We also need the "blas" package. Otherwise we get reports
+    # of "cblas.h: No such file or directory" at import time.
+    # Ref: <https://github.com/conda-forge/aesara-feedstock/issues/80>
+    - blas
+    - wheel
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.14
+    - filelock
+    - etuples
+    - logical-unification
+    - miniKanren
+    - cons
+    - typing_extensions
+    - mkl-service  # [not ((osx and x86_64) or linux and (aarch64 or ppc64le))]
+    - openblas     # [(osx and x86_64) or linux and (aarch64 or ppc64le)]
```

Comment:
`setuptools` is required in runtime too https://github.com/search?q=repo%3Apymc-devs%2Fpytensor%20setuptools&type=code

Reply:
thanks for catching that!